content/hi/debugging.md

```diff
@@ -0,0 +1,24 @@
+---
+title: डिबगिंग(Debugging)
+status: Feedback Appreciated
```

Comment:
please check: [https://github.com/cncf/glossary/blob/main/content/en/debugging.md](https://github.com/cncf/glossary/blob/main/content/en/debugging.md)

the status seems to have changed. Make sure your main branch is latest y running 'git pull'. you may then rebase this branch with main

Reply:
I changed status. You can review it. @Garima-Negi